tests/framework/PostProcessors/Validation/test_validation_gate_pcm_SETH.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="all">
+  <RunInfo>
+    <WorkingDir>PhysicsGuidedCoverageMapping</WorkingDir>
+    <Sequence>load,PCM</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/Validation/test_validation_gate_pcm_SETH</name>
+    <author>syin</author>
+    <created>2022-09-09</created>
+    <classesTested>PostProcessors.Validation.PhysicsGuidedCoverageMapping</classesTested>
+    <description>
+        This test is aimed to show how PCM works.
+        This test is using SETH-C and SETH-D data 
+        as experiment (Feature) and application (Target) models.
+        Here, three timesteps' samples from SETH-C are used as experiment responses (Features),
+        one timestep's samples from SETH-D are used as application responses (Target)
+        The output is a fraction value reflecting the uncertainty reduction fraction 
+        using Feature to validate Target comparing to the Target prior.
+        The output name convention is 'pri_post_stdReduct_'+"Target name".
+    </description>
+    <revisions>
+      <revision author="syin" date="2022-10-03">...</revision>
```

Comment:
please update this if you have revisions

Reply:
The revision date has been modified and this test file name has been changed according to CamelBack format.